crates/requirements-txt/src/lib.rs

```diff
@@ -1254,6 +1249,27 @@ mod test {
         insta::assert_debug_snapshot!(snapshot, actual);
     }
 
+    #[cfg(unix)]
+    #[test_case(Path::new("bare-url.txt"))]
```

Comment:
I really tried, but there are so many weirdnesses in the Windows version because it's a file URL, or something. I gave up with the filters.